modules/tinymce/src/plugins/quickbars/demo/html/demo.html

```diff
@@ -36,8 +36,8 @@ <h2>The world’s first rich text editor in the cloud</h2>
 
 
         <p>
-          One of these enhancements is <strong>Tiny Drive</strong>: imagine file management for TinyMCE, in the cloud, made super easy.
-          Learn more at <a href='https://www.tiny.cloud/tinydrive/'>tiny.cloud/tinydrive</a>, where you’ll find a working demo and an opportunity to provide feedback to the product team.
+          One of these enhancements is <strong>Drive</strong>: imagine file management for TinyMCE, in the cloud, made super easy.
```

Comment:
isn't the product name still `Tiny Drive`?

Reply:
no thats since been changed https://www.tiny.cloud/drive/

Reply:
That page uses both so I don't think it's a good source of truth:
![image](https://user-images.githubusercontent.com/1575550/155907902-a59618f8-b9e7-4c5f-abfe-f73e2e0becdc.png)

Reply:
Realised this is a demo page, so wording isn't critical. Therefore, resolving.